frontend/src/metabase/query_builder/components/filters/modals/BulkFilterList/BulkFilterList.styled.tsx

```diff
@@ -10,7 +11,12 @@ export const ListRoot = styled.div`
 export const ListRow = styled.div`
   display: grid;
   grid-template-columns: minmax(0, 1fr) minmax(0, 2fr);
-  padding: 0.375rem 0;
+  gap: ${space(2)};
+  padding: 1.5rem 2rem;
+  border-bottom: 1px solid ${color("border")};
+  &:last-child {
+    border-bottom: none;
+  }
```

Comment:
this puts half-width borders between filters on the same field

![Screen Shot 2022-07-08 at 11 09 04 AM](https://user-images.githubusercontent.com/30528226/178038676-53e55080-ea27-4814-b298-4e289d32a828.png)